libraries/kmp-iap/example/composeApp/build.gradle.kts

```
@@ -21,6 +22,23 @@ fun loadEnvProperties(): Properties {
 
 val envProperties = loadEnvProperties()
 
+// Read library version mode from libraries-versions.jsonc
+val kmpIapMode: String = try {
+    val librariesVersions = JsonSlurper().parseText(
+        rootProject.file("../../libraries-versions.jsonc").readText()
+            .replace(Regex("^\\s*//.*$", RegexOption.MULTILINE), "")
+    ) as Map<*, *>
+    librariesVersions["kmp-iap"]?.toString() ?: "local"
+} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)